segments/hostname.sh

```diff
@@ -3,24 +3,33 @@
 
 TMUX_POWERLINE_SEG_HOSTNAME_FORMAT="${TMUX_POWERLINE_SEG_HOSTNAME_FORMAT:-short}"
 
+# check if user has hostname command
+has_hostname() {
+  command -v $1 >/dev/null
+}
+
 generate_segmentrc() {
-	read -r -d '' rccontents <<EORC
```

Comment:
Could you please update the PR to not change the indentation. I think it was already correct (hard tabs and not spaces). The project should be consistent, and PRs becomes more easy to follow if they focus on functional changes :)

Reply:
This is still needed :)